python_modules/dagster/dagster/_core/definitions/decorators/asset_decorator.py

```diff
@@ -107,16 +103,11 @@ def asset(
         name (Optional[str]): The name of the asset.  If not provided, defaults to the name of the
             decorated function. The asset's name must be a valid name in dagster (ie only contains
             letters, numbers, and _) and may not contain python reserved keywords.
-        namespace (Optional[Sequence[str]]): **Deprecated (use `key_prefix`)**. The namespace that
-            the asset resides in.  The concatenation of namespace and name forms the asset key. Each
-            item in namespace must be a valid name in dagster (ie only contains letters, numbers,
-            and _) and may not contain python reserved keywords.
         key_prefix (Optional[Union[str, Sequence[str]]]): If provided, the asset's key is the
             concatenation of the key_prefix and the asset's name, which defaults to the name of
             the decorated function. Each item in key_prefix must be a valid name in dagster (ie only
             contains letters, numbers, and _) and may not contain python reserved keywords.
-        ins (Optional[Mapping[str, AssetIn]]): A dictionary that maps input names to their metadata
-            and namespaces.
+        ins (Optional[Mapping[str, AssetIn]]): A dictionary that maps input names to their metadata.
```

Comment:
a nitpick, because this wasn't introduced in this commit, but  "A dictionary that maps input names to their metadata" is not exactly accurate. If we steal the description of the op's "ins" parameter, that would look like "Information about the inputs to the asset. " (not very descriptive, but at least it's symmetrical with the op name)

feel free to ignore

Reply:
👍